src/components/ProductCard.jsx

```diff
@@ -0,0 +1,30 @@
+const ProductCard = ({id,title, isLiked, imageUrl, qty, price, rating, storeName, category}) => {
+  return (
+    <div className="flex-center">
+      <div className="card card-shadow p-1">
+        <div className="image">
+          <img src={imageUrl} alt={title} />
+        </div>
+        <div className="card-heading-container">
+          <h3 className="font-md font-medium">{storeName}</h3>
+          <h5 className="font-sm font-light p-b-1">{title}</h5>
+        </div>
+        <div className="card-badge card-badge-position">{category}</div>
+        <div className="favourite-icon favourite-icon-position">
+          <a className="btn btn-link-noBorder" href>
```

Comment:
use link tag here

Reply:
will look into it thanks for the suggestion.